llvm/lib/Target/SBF/SBFRegisterInfo.cpp

```diff
@@ -166,9 +166,9 @@ int SBFRegisterInfo::resolveInternalFrameIndex(
       dbgs() << "Error: A function call in method "
              << MF.getFunction().getName()
              << " overwrites values in the frame. Please, decrease stack usage "
-             << "or remove parameters from the call.\n\n";
-      report_fatal_error(
-          "The function call may cause undefined behavior during execution.");
+             << "or remove parameters from the call."
+             << "The function call may cause undefined behavior "
+                "during execution.\n\n";
```

Comment:
Just a note that in the previous message you use the word "unexpected" v. "undefined" in this message, w.r.t. behavior.

Reply:
Let me switch everything to `undefined`.